mk/doc-gen.py

```diff
@@ -18,6 +19,10 @@ def convert_path(url, base_url, after):
 
 
 if __name__ == '__main__':
+    if len(sys.argv) < 2:
```

Comment:
is there a README.md somewhere that needs to be updated to reflect this?

Reply:
no, the README.md just states to call the make target